wiki/markdown/REST-API.md

```diff
@@ -5,7 +5,7 @@ the following endpoints:
 
 
 * [Query](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query.md) - query Eiffel Intelligence for aggregated objects which fulfills a given condition.
-* [Query](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query.md) - query for missed notifications which fulfills a given condition.
+* [FailedNotification](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query.md) - query for failed notifications which fulfills a given condition.
```

Comment:
Add space: FailedNotification --> Failed notifications

I actually would like the paragraph about querying missed notifications (like it's named today) to be moved from this file https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query.md to a separate one containing only the information about the /failed-notificationss endpoint. Then this link needs to be updated.

Reply:
done

Reply:
Remember to update the links in README.md and wiki/site.xml also